workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java

```diff
@@ -206,7 +220,7 @@ public void testRemoveUsersFromProject_WithEmptyUsersList() throws Exception {
 
 	ProjectResponseDTO createSampleProject(String name) {
 		ProjectResponseDTO responseDTO = new ProjectResponseDTO();
-		responseDTO.setId(UUID.randomUUID());
+		responseDTO.setId(UUID.fromString("af7c1fe6-d669-414e-b066-e9733f0de7a8"));
```

Comment:
Some tests use this method.
WDYT to set the "hard coded" `Id` in  `testListProjects_when_etagEquals_then_return_notModified` test instead?

Reply:
I was expecting the hash of the same UUID string should always return the same value. but different java version may have different result, I will rollback to randomUUID

Reply:
If we want to track this, maybe we can add another unit tests. So we can be sure that each generated id will not change